tests/server/wildfly/src/test/java/org/hawkular/apm/tests/wildfly/AnalyticsServiceRESTTest.java

```diff
@@ -179,6 +179,46 @@ public void testGetBoundEndpoints() {
     }
 
     @Test
+    public void testGetTransactionInfo() {
+        Trace trace1 = new Trace();
+        trace1.setId("1");
+        trace1.setBusinessTransaction("trace1");
+        trace1.setStartTime(System.currentTimeMillis() - 4000); // Within last hour
+
+        Component c1 = new Component();
+        c1.getProperties().add(new Property("prop1", "value1"));
+        trace1.getNodes().add(c1);
+
+        Trace trace2 = new Trace();
+        trace2.setId("2");
+        trace2.setBusinessTransaction("trace2");
+        trace2.setStartTime(System.currentTimeMillis() - 2000); // Within last hour
+
+        Component c2 = new Component();
+        c2.getProperties().add(new Property("prop2", "value2"));
+        trace2.getNodes().add(c2);
+
+        try {
+            publisher.publish(null, Arrays.asList(trace1, trace2));
+        } catch (Exception e1) {
+            fail("Failed to store: " + e1);
+        }
+
+        // Wait to ensure record persisted
+        Wait.until(() -> service.searchFragments(null, new Criteria()).size() == 2);
+
+        // Wait to result derived
+        Wait.until(() -> analytics.getTraceCompletionTimes(null, new Criteria()).size() == 2);
+
+        List<TransactionInfo> tis = analytics.getTransactionInfo(null, new Criteria());
+
+        assertNotNull(tis);
+        assertEquals(2, tis.size());
+        assertTrue(tis.get(0).getName().equals("trace1"));
```

Comment:
Minor thing, In this case it's maybe better to use `Assert.assertEquals(expected, actual)` .